docs/release-notes.md

```diff
@@ -0,0 +1 @@
+--8<-- "CHANGELOG.md"
```

Comment:
🤔

Reply:
With this we have the CHANGELOG on the root level, and we reference it from the documentation, and we don't duplicate the content.